cli/command/config/inspect.go

```diff
@@ -52,7 +52,7 @@ func RunConfigInspect(dockerCli command.Cli, opts InspectOptions) error {
 	// check if the user is trying to apply a template to the pretty format, which
 	// is not supported
 	if strings.HasPrefix(f, "pretty") && f != "pretty" {
-		return fmt.Errorf("Cannot supply extra formatting options to the pretty template")
+		return errors.New("cannot supply extra formatting options to the pretty template")
```

Comment:
Slightly concerned about this commit, as it changes the output of errors. For sure, people "shouldn't" string match these, but it's _possible_, and may be unexpected in a patch release.

Reply:
That's fair; I'll replace that commit with a golangci-lint exclude rule